src/main/java/com/staxrt/tutorial/controller/UserController.java

```diff
@@ -0,0 +1,131 @@
+/*
+ *
+ *  Copyright (c) 2018-2020 Givantha Kalansuriya, This source is a part of
+ *   Staxrt - sample application source code.
+ *   http://staxrt.com
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+package com.staxrt.tutorial.controller;
+
+import com.staxrt.tutorial.exception.ResourceNotFoundException;
+import com.staxrt.tutorial.model.User;
+import com.staxrt.tutorial.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * The type User controller.
+ *
+ * @author Givantha Kalansuriya
+ */
+@RestController
+@RequestMapping("/api/v1")
+public class UserController {
```

Comment:
Proponowałbym nie robić obsługi użytkowników, żeby mieć klarowność co jest w tej zmianie. Nie za bardzo rozumiem jaki zakres ma aktualna obsługa użytkowników? Co się da a czego się jeszcze nie da itp. Lepsza mniejsza ale jakaś zamknięta całość niż mieszanie biznesowych funkcjonalności.

Reply:
Można dodawać / usuwać / wyświetlać / modyfikować użytkowników to samo co dla post

Reply:
Za chwilę będę tego używał tak więc mogę to wyrzucić z pull requesta i commita żeby biznesowo wersja V1 zajmowała się Tylko i wyłącznie postami

Reply:
Ja bym wyrzucił użytkowników, bo jak się chwilę zastanowisz to chyba nie do końca będzie to samo API. Jedno API będzie pewnie dla aplikacji klienckich (w tym aplikacja webowa), ale nie chciałbyś umożliwić użytkownikowi dodanie/usunięcie innego użytkownika. To będzie już raczej api dla części "backoffice". Użytkownik wolałbyś, żeby mógł się zarejestrować a potem zalogować za pomocą aplikacji klienckiej.

Reply:
review